localization/english/wc_buildings_l_english.yml

```diff
@@ -269,3 +275,632 @@
+ 
+ building_obelisk_of_the_sun_01:1 "Obelisk of the Sun"
+ building_obelisk_of_the_sun_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
+ building_obelisk_of_the_moon_01:1 "Obelisk of the Moon"
+ building_obelisk_of_the_moon_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
+ building_obelisk_of_the_stars_01:1 "Obelisk of the Stars"
+ building_obelisk_of_the_stars_01_desc:1 "One of three obelisks in Uldum that were once used to mask the area from the outside world."
+ 
+ building_type_northshire_abbey_01:0 "Northshire Abbey"
+ building_type_northshire_abbey_01_desc:1 "Home to the Brotherhood of Northshire. Its Clerics supported the citizens of the kingdom and ensured the protection of the realm alongside the Stormwind Army for decades."
+ building_northshire_abbey_01:1 "Northshire Abbey"
+ building_northshire_abbey_01_desc:1 "$building_type_northshire_abbey_01_desc$"
+ building_type_scarlet_monastery_01:0 "Old Monastery"
+ building_type_scarlet_monastery_01_desc:1 "A proud bastion of Lordaeron's priesthood - a center for learning and enlightenment."
+ building_type_scarlet_monastery_02:0 "Scarlet Monastery"
+ building_type_scarlet_monastery_02_desc:1 "A stronghold for the Scarlet Crusade."
+ building_scarlet_monastery_01:1 "Old Monastery"
+ building_scarlet_monastery_01_desc:1 "$building_type_scarlet_monastery_01_desc$"
+ building_scarlet_monastery_02:1 "Scarlet Monastery"
+ building_scarlet_monastery_02_desc:1 "$building_type_scarlet_monastery_02_desc$"
+ building_type_shado_pan_monastery_01:0 "Shado-Pan Monastery"
+ building_type_shado_pan_monastery_01_desc:1 "This mountain retreat serves as the home and training grounds of the mysterious Shado-Pan, a pandaren faction dedicated to protecting Pandaria."
+ building_shado_pan_monastery_01:1 "Shado-Pan Monastery"
+ building_shado_pan_monastery_01_desc:1 "$building_type_shado_pan_monastery_01_desc$"
+ building_type_tian_monastery_01:0 "Tian Monastery"
+ building_type_tian_monastery_01_desc:1 "The monks here specialize in teaching young pandaren how to fight to prevent ever becoming slaves to the mogu again. Many great fighters trained at this temple such as Yu Ming the Blade, Liang the Unbreakable, and even the great Emperor Shaohao."
+ building_tian_monastery_01:1 "Tian Monastery"
+ building_tian_monastery_01_desc:1 "$building_type_tian_monastery_01_desc$"
+ 
+ building_type_gallywix_pleasure_palace_01:0 "Gallywix Pleasure Palace"
+ building_type_gallywix_pleasure_palace_01_desc:0 "While staying in one lavishly luxuriant castle is fine, having a few more to choose between never hurts."
+ building_gallywix_pleasure_palace_01:0 "Chateau"
+ building_gallywix_pleasure_palace_01_desc:0 "When the burdens of responsibility become too much (or an angry mob of peasants is storming the gates), there's no place like the rural, far-away chateau for some impromptu relaxation!"
```

Comment:
```suggestion
 building_gallywix_pleasure_palace_01_desc:0 "When the burdens of responsibility become too much (or an angry mob of peasants is storming the gates), there's no place like the rural, far-away chateau for some impromptu relaxation."
```

Reply:
This one was a copied quote from the wiki